src/openrct2-ui/windows/TrackDesignPlace.cpp

```diff
@@ -127,9 +127,9 @@ static void window_track_place_draw_mini_preview_track(
     TrackDesign* td6, int32_t pass, CoordsXY origin, CoordsXY& min, CoordsXY& max);
 static void window_track_place_draw_mini_preview_maze(
     TrackDesign* td6, int32_t pass, CoordsXY origin, CoordsXY& min, CoordsXY& max);
-static LocationXY16 draw_mini_preview_get_pixel_position(int16_t x, int16_t y);
-static bool draw_mini_preview_is_pixel_in_bounds(LocationXY16 pixel);
-static uint8_t* draw_mini_preview_get_pixel_ptr(LocationXY16 pixel);
+static CoordsXY draw_mini_preview_get_pixel_position(CoordsXY pos);
```

Comment:
See first remark.

Reply:
I've dropped the whole commit on `TrackDesignPlace´